code/modules/jobs/job_types/enclave.dm

```diff
@@ -113,13 +113,13 @@
 		/obj/item/grenade/f13/plasma = 2,
 		)
 
-/datum/outfit/loadout/lt_rhino
+/datum/outfit/loadout/lt_rhino //Main focus of the loadout is actually the revolver, This is a loadout originally intended to be specific to me, regardless. I decided against it as it would be best everyone get to use it
```

Comment:
Either include a name at the end of the comment, so it can be referenced in the future, or remove the comment altogether.